gems/sorbet-runtime/test/types/final_method.rb

```diff
@@ -0,0 +1,319 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+class Opus::Types::Test::FinalMethodTest < Critic::Unit::UnitTest
+  before do
+    T::Configuration.enable_final_checks_for_include_extend
```

Comment:
I believe putting this here makes it global for the entire test suite, and we never reset it. Can we either

- put this in the test_helper, so that it's more clear this is global and can't be reset
- make it possible to reset whether `Module` has been monkey patched, so that we can enable the monkey patch only in certain tests

Look at the other `T::Configuration` tests for how they set up and tear down individual T::Configuration tests.